cli-requirejs-demo/aurelia_project/aurelia.json

```diff
@@ -122,57 +107,10 @@
               "multiple-select.css"
             ]
           },
-          {
-            "name": "aurelia-slickgrid",
-            "path": "../node_modules/aurelia-slickgrid/dist/amd",
-            "main": "index",
-            "resources": [
-              "**/*.{css,html}"
-            ]
-          },
-          {
-            "name": "slick-gridmenu",
-            "path": "../node_modules/slickgrid/controls",
-            "main": "slick.gridmenu.js"
-          },
           {
```

Comment:
since jquery-ui-dist is nowhere requested by itself this makes sure its eagerly configured and bundled